Doc/library/tkinter.rst

```diff
@@ -67,16 +67,37 @@ Or, more often::
    from tkinter import *
 
 
-.. class:: Tk(screenName=None, baseName=None, className='Tk', useTk=1)
-
-   The :class:`Tk` class is instantiated without arguments. This creates a toplevel
-   widget of Tk which usually is the main window of an application. Each instance
-   has its own associated Tcl interpreter.
-
-   .. FIXME: The following keyword arguments are currently recognized:
-
-
-.. function:: Tcl(screenName=None, baseName=None, className='Tk', useTk=0)
+.. class:: Tk(screenName=None, baseName=None, className='Tk', useTk=True, sync=False, use=None)
+
+   Toplevel Tk widget, which is usually the main window of an application.
+   Each instance has its own associated Tcl interpreter.
+
+   The :class:`Tk` class is typically instantiated using all default values.
+   However, the following keyword arguments are currently recognized:
+
+   *screenName*
+      Sets the :envvar:`DISPLAY` environmental variable. (X11 only)
```

Comment:
Does Tk() really set the environment variable? C code:

```
    if (screenName != NULL)
        Tcl_SetVar2(v->interp, "env", "DISPLAY",
                    screenName, TCL_GLOBAL_ONLY);
```

It seems to be a Tcl variable, not an environment variable like C putenv() or Python os.environ, no?

Reply:
It says it does - https://www.tcl.tk/man/tcl8.5/TclCmd/tclvars.htm#M4.

Reply:
I tested and I confirm that it sets the env var, ok :-)

```
>>> import os, tkinter
>>> del os.environ['DISPLAY']
>>> tkinter.Tk(screenName=':1')
<tkinter.Tk object .>

>>> import ctypes.util
>>> libc=ctypes.cdll.LoadLibrary(ctypes.util.find_library('c'))
>>> libc.getenv.restype=ctypes.c_char_p
>>> libc.getenv(b'DISPLAY')
b':1'
```